tests_v2/test_workspace_create_layout.py

```
@@ -0,0 +1,283 @@
+        _must(_pane_count(c, ws) == 2, f"expected 2 panes, got {_pane_count(c, ws)}")
+        surfaces = _surface_list(c, ws)
+        _must(len(surfaces) == 2, f"expected 2 surfaces, got {len(surfaces)}")
+        types = {str(s.get("type")) for s in surfaces}
+        _must(types == {"terminal"}, f"expected all terminal surfaces, got {types}")
+        c.select_workspace(baseline)
+    finally:
+        if ws:
+            try:
+                c.close_workspace(ws)
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)